src/terminal/input/terminalInput.hpp

```diff
@@ -34,11 +34,16 @@ namespace Microsoft::Console::VirtualTerminal
         ~TerminalInput() = default;
 
         bool HandleKey(const IInputEvent* const pInEvent) const;
+        bool HandleChar(const char16_t ch);

```

Comment:
Why `char16_t` instead of `wchar_t`? 

I don't think we use `char16_t` anywhere else in the code. Is there a good reason for one over the other?

Reply:
I did it this way because the method that I extracted this code from used `const auto ch = e.Character();` which ended up being a `char16_t`, however it seems that it was assumed to act as a `wchar_t` anyways so I'll just convert it.